MSAL/src/configuration/MSALPublicClientApplicationConfig.m

```diff
@@ -130,6 +130,7 @@ - (id)copyWithZone:(NSZone *)zone
     item->_verifiedRedirectUri = [_verifiedRedirectUri copyWithZone:zone];
     item->_extraQueryParameters = [_extraQueryParameters copyWithZone:zone];
     item->_multipleCloudsSupported = _multipleCloudsSupported;
+    item->_bypassRedirectURIValidation = _bypassRedirectURIValidation;
```

Comment:
I'll do it in the other PR, that I opened with changes into MSAL Objective-C. This code is here because it was merged from the other working branch, so you guys would be able to test the feature.